lib/src/firebase_cache_manager/mobile_firebase_cache_manager.dart

```diff
@@ -197,4 +197,12 @@ class FirebaseCacheManager extends BaseFirebaseCacheManager {
   Future<String> getFullLocalPath(String fileName) async {
     return join(await _cacheDirectoryPath, fileName);
   }
+
+  @override
+  Future<bool> isCached(FirebaseUrl firebaseUrl) async {
+    final manager = await _cacheManager;
+    final cachedObject = await manager.get(firebaseUrl.uniqueId);
+
+    return cachedObject != null;
```

Comment:
We also need to check whether the file exists in the device storage.  So can you please add that condition here too?